x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/dimension_editor.scss

```diff
@@ -22,6 +22,10 @@
   }
 }
 
+.lnsIndexPatternDimensionEditor__section:empty {
+  display: none;
+}
```

Comment:
Out of curiosity, under what circumstances are we rendering empty `.lnsIndexPatternDimensionEditor__section` elements? Rather than using CSS to hide it, would it be better to prevent the rendering of this element if it's empty?

Reply:
It's because the [conditions](https://github.com/elastic/kibana/blob/main/x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/dimension_editor.tsx#L434) that actually put the children of this element  in place are multiple and quite complex so I just wanted an easy fix without having to modify a lot of code.